### ISSUE TITLE
Failed to start suite. Error: Request failed with status code 404

### ISSUE THREAD

**Issue by DHenry7471** [reportportal/agent-js-playwright#57]
Getting this error whenever I try to run the example playwright tests with ReportPortal

`failed to finish test. Error: Request failed with status code 404
    at /XXXXXX/playwright/node_modules/.pnpm/@reportportal+client-javascript@5.0.6/node_modules/@reportportal/client-javascript/lib/rest.js:37:15`

Here's my `config.ts `file
```
const RPconfig = {
  'token': 'XXXXXXXXXXXXX',
  'endpoint': 'https://localhost:8080/api/v1',
  'project': 'XXXXXXXXXl',
  'launch': 'superadmin_TEST_EXAMPLE',
  'attributes': [
    {
      'key': 'key',
      'value': 'value',
    },
    {
      'value': 'value',
    },
  ],
  'description': 'Playwright E2E tests',
};

const config: PlaywrightTestConfig = {
  testDir: './tests',
  /* Maximum time one test can run for. */
  timeout: 30 * 1000,
  expect: {
    /**
     * Maximum time expect() should wait for the condition to be met.
     * For example in `await expect(locator).toHaveText();`
     */
    timeout: 5000
  },
  /* Run tests in files in parallel */
  fullyParallel: true,
  /* Fail the build on CI if you accidentally left test.only in the source code. */
  forbidOnly: !!process.env.CI,
  /* Retry on CI only */
  retries: process.env.CI ? 2 : 0,
  /* Opt out of parallel tests on CI. */
  workers: process.env.CI ? 1 : undefined,
  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
  reporter: [['@reportportal/agent-js-playwright', RPconfig]],
  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
  use: {
    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
    actionTimeout: 0,
    /* Base URL to use in actions like `await page.goto('/')`. */
    // baseURL: 'http://localhost:3000',

    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
    trace: 'on-first-retry',
  },
```

**Comment by AmsterGet** [comment]
Hello @DHenry7471 !
Did you use the example from our [repo](https://github.com/reportportal/examples-js) with examples?
Could you please provide a reproducible example for this issue?

**Comment by DHenry7471** [comment]
> Hello @DHenry7471 ! Did you use the example from our [repo](https://github.com/reportportal/examples-js) with examples? Could you please provide a reproducible example for this issue?

I was using the example tests generated from playwright itself

I'm also seeing this error as well
```
/XXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:462
                        rootSuiteName = parentObj.rootSuite;
                                                  ^

TypeError: Cannot read properties of undefined (reading 'rootSuite')
    at RPReporter.<anonymous> (/XXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/src/reporter.ts:483:37)
    at step (/XXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:60:23)
    at Object.next (/XXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:41:53)
    at /XXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:35:71
    at new Promise (<anonymous>)
    at __awaiter (/XXXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:31:12)
    at RPReporter.onTestEnd (/XXXXX/test/playwright/node_modules/.pnpm/@reportportal+agent-js-playwright@5.0.5/node_modules/@reportportal/agent-js-playwright/build/reporter.js:423:16)
    at /XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/reporters/multiplexer.js:69:131
    at wrap (/XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/reporters/multiplexer.js:119:5)
    at Multiplexer.onTestEnd (/XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/reporters/multiplexer.js:66:45)
    at Dispatcher._reportTestEnd (/XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/dispatcher.js:574:159)
    at Worker.onTestEnd (/XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/dispatcher.js:240:12)
    at Worker.emit (node:events:527:28)
    at ChildProcess.<anonymous> (/XXXXX/test/playwright/node_modules/.pnpm/@playwright+test@1.25.0/node_modules/@playwright/test/lib/dispatcher.js:633:12)
```

**Comment by AmsterGet** [comment]
Hello guys!
We have released a version 5.0.6 with a fix.
Could you confirm that this fixes the issue?

**Comment by AmsterGet** [comment]
Closed together with https://github.com/reportportal/agent-js-playwright/issues/56

@DHenry7471 feel free to reopen it if needed.
